### PR TITLE
7089 header issue adc export

### PR DESCRIPTION
**Pull request by siddharthsatyakam**
Tested the changes in [http://sable:8080/job/ADC_multi_project_export/1550/console]

and also in the ticket: https://bmi-work.kumc.edu/work/ticket/7089

### CODE REVIEW

File: adc_export.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def main(get_config,
         if field_names == ('',):
             field_names = (data_proj.def_field,) + field_names
 
+        print(field_names)
```
Comment (lav-patel) [inline adc_export.py]: Do you still need this print? if not, remove it.

File: adc_export.py [hunk 2]
```diff
@@ -51,11 +52,12 @@ def main(get_config,
             log.info('Records: %s', records)
             for record_chunk in chunks(records, chunk_size):
                 log.info('Chunk: %s to %s', record_chunk[0], record_chunk[-1])
+
```
Comment (lav-patel) [inline adc_export.py]: Please remove empty line

File: adc_export.py [hunk 3]
```diff
@@ -51,11 +52,12 @@ def main(get_config,
...
                 data = data_proj.export_records(records=record_chunk,
                                                 format=file_format,
                                                 forms=[row['formname'], ],
-                                                fields=field_names,
                                                 event_name='unique')
+                #print(data)
```
Comment (lav-patel) [inline adc_export.py]: Remove commented our print

File: adc_export.py [hunk 4]
```diff
@@ -51,11 +52,12 @@ def main(get_config,
             log.info('Records: %s', records)
             for record_chunk in chunks(records, chunk_size):
                 log.info('Chunk: %s to %s', record_chunk[0], record_chunk[-1])
+
                 data = data_proj.export_records(records=record_chunk,
                                                 format=file_format,
                                                 forms=[row['formname'], ],
-                                                fields=field_names,
```
Comment (lav-patel) [inline adc_export.py:57]: Can you give me directly link to the exported csv(s)  ? And does it have expected output?
example:

http://sable:8080/job/ADC_multi_project_export/xyz_1.csv
http://sable:8080/job/ADC_multi_project_export/xyz_2.csv

Comment (siddharthsatyakam) [reply adc_export.py:57]: http://sable:8080/job/ADC_multi_project_export/ws/

Yes from a scan everything is being displayed good in csv excels

File: adc_export.py (hunk 1 above)
Comment (siddharthsatyakam) [reply adc_export.py]: Done

File: adc_export.py (hunk 2 above)
Comment (siddharthsatyakam) [reply adc_export.py]: Done

File: adc_export.py (hunk 3 above)
Comment (siddharthsatyakam) [reply adc_export.py]: Done

File: adc_export.py [hunk 5]
```diff
@@ -50,11 +50,10 @@ def main(get_config,
             header_written = False
             log.info('Records: %s', records)
             for record_chunk in chunks(records, chunk_size):
-                log.info('Chunk: %s to %s', record_chunk[0], record_chunk[-1])
+                log.info('Chunk: %s to %s', record_chunk[0], record_chunk[-1])
```
Comment (lav-patel) [inline adc_export.py]: Please remove whitespace at the end

Comment (siddharthsatyakam) [reply adc_export.py]: DONE

**Review by lav-patel — APPROVED** [review]

**Pull request merged**